src/test/main_tests.cpp

```diff
@@ -13,54 +13,6 @@
 
 BOOST_FIXTURE_TEST_SUITE(main_tests, TestingSetup)
 
-static void TestBlockSubsidyHalvings(const Consensus::Params& consensusParams)
-{
-    // tested in dogecoin_tests.cpp
-    //int maxHalvings = 64;
-    //CAmount nInitialSubsidy = 50 * COIN;
-
-    //CAmount nPreviousSubsidy = nInitialSubsidy * 2; // for height == 0
-    //BOOST_CHECK_EQUAL(nPreviousSubsidy, nInitialSubsidy * 2);
-    //for (int nHalvings = 0; nHalvings < maxHalvings; nHalvings++) {
-    //    int nHeight = nHalvings * consensusParams.nSubsidyHalvingInterval;
-    //    CAmount nSubsidy = GetBlockSubsidy(nHeight, consensusParams);
-    //    BOOST_CHECK(nSubsidy <= nInitialSubsidy);
-    //    BOOST_CHECK_EQUAL(nSubsidy, nPreviousSubsidy / 2);
-    //    nPreviousSubsidy = nSubsidy;
-    //}
-    //BOOST_CHECK_EQUAL(GetBlockSubsidy(maxHalvings * consensusParams.nSubsidyHalvingInterval, consensusParams), 0);
-}
```

Comment:
Looks good:

Variation in method signature has also been covered by the search (see `int nSubsidyHalvingInterval`)

![image](https://user-images.githubusercontent.com/4400555/119738150-27527380-be78-11eb-851e-39b5c9c15b4d.png)